notebooks/03_embedding_graph_structured_data_h2.ipynb

```diff
@@ -45,13 +45,22 @@
   },
```

Comment:
<p>NIT: Predicted Group 1, Predicted Group 2 (what do you think, @cschewmake2?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>